fusor-ember-cli/app/components/base-f.js

```diff
@@ -16,11 +16,11 @@ export default Ember.Component.extend({
   },
 
   labelClassSize: Ember.computed(function () {
-    return this.getWithDefault('labelSize', 'col-lg-2 col-md-3 col-sm-5 col-xs-4 col-xs-offset-1');
+    return this.getWithDefault('labelSize', 'col-lg-2 col-md-3 col-sm-5');
   }),
 
   inputClassSize: Ember.computed(function () {
-    return this.getWithDefault('inputSize', 'col-lg-4 col-md-6 col-sm-7 col-xs-4');
+    return this.getWithDefault('inputSize', 'col-lg-4 col-md-6 col-sm-6');
```

Comment:
changing sm-7 to sm-6 fixes the issue. I'm not sure what is causing extra padding to cause a row sm-12 to wrap.
removing the `xs` classes causes the labels / inputs to wrap vertically which looks better rather than forcing label / input to be on the same row.